packages/amplify-python-function-runtime-provider/src/util/pyUtils.ts

```diff
@@ -13,8 +14,7 @@ export async function getPipenvDir(srcRoot: string): Promise<string> {
     throw new Error(`Could not find 'python3' or 'python' executable in the PATH.`);
   }
 
-  const pyVersion = await execAsStringPromise(`${pyBinary} --version`);
-  let pipEnvPath = path.join(pipEnvDir, 'lib', 'python' + majMinPyVersion(pyVersion), 'site-packages');
+  let pipEnvPath = path.join(pipEnvDir, 'lib', `python${getPipfilePyVersion(path.join(srcRoot, 'Pipfile'))}`, 'site-packages');
```

Comment:
Would there be any benefit to falling back to `majMinPyVersion()` if `getPipfilePyVersion()` fails?

Reply:
If there's no version specified in the Pipfile, creating the virtual env wont even work so I don't think we need a fallback here